### DISCUSSION TITLE
How to enable SPDX feature

### DISCUSSION THREAD

**Discussion by Sa2shi** [eclipse-sw360/sw360#2443] [Q&A]
Hello, 

I just built the sw360 instance with the following instruction.
https://github.com/eclipse-sw360/sw360/blob/main/README_DOCKER.md

However, when I tried the following instruction by entering the sw360 container, it didn't work well.
https://eclipse.dev/sw360/docs/userguide/spdx_document/#how-to-enable-this-feature

- [Step1] Chekcing the sw360 container id with the "docker ps" command
- [Step2]  Entering the sw360 container with the "docker exec -it <container id> bash"
- [Step3] Adding config spdx.document.enabled = true (/etc/sw360/sw360.properties)
- [Step4] Restarting contaiers by "docker-compose stop" and "docker-compose up" commands.

Do you have any idea to solve this issue?

The sw360 version is sw360-17.0.0-M1, and the commit id is https://github.com/eclipse-sw360/sw360/commit/ae997be20a95f3afa253969762fb502b1364e489.

Thanks,

**Comment by KoukiHama** [comment]
I will check it

**Reply by KoukiHama** [reply]
SPDX feature is not available on version 17.0.0. So it cannot be used.

you may can do test SPDX features by this one.
https://github.com/eclipse-sw360/sw360/releases/tag/sw360-18.0.1-M1

**Reply by Sa2shi** [reply]
Thank you, and sorry for my late reply.
I got it. I'm going to try it!
